druntime/src/importc.h

```diff
@@ -164,3 +164,8 @@
 #if __APPLE__
 #undef __SIZEOF_INT128__
 #endif
+
+// Wide-char compatibility
+#define wchar_t wchar
```

Comment:
this is wrong, see https://github.com/dlang/dmd/blob/master/druntime/src/core/stdc/stddef.d#L35-L40